src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheckTest.java

```diff
@@ -75,7 +75,7 @@ public void testCountEmpty() throws Exception {
         checkConfig.addAttribute("max", "19");
         checkConfig.addAttribute("countEmpty", "false");
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputMethodLengthSimple.java"), expected);
+        verify(checkConfig, getPath("InputMethodLengthCountEmpty.java"), expected);
```

Comment:
it seems like name of the test contradicts to what it does: `testCountEmpty` and `countEmpty = false`

lets change test name and input name to
`testCountEmptyIsFalse`  `InputMethodLengthCountEmptyIsFalse`

what do you think?

Reply:
On it.

Reply:
Done.